### ISSUE TITLE
adding alt tag option

### ISSUE THREAD

**Issue by vedranvucic** [vikdiesel/justboil.me#27]
Hello, I think that upload interface should have option to add alt tag description for image since it is requirement for accessibility of images for persons with disabilities. Many legislations require that as mandatory to each image so the persons with disabilities can use screen readers while going over images.  Thanks
